mariadb/templates/configmaps/seed.sh.yaml.j2

```diff
@@ -0,0 +1,120 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  seed.sh: |+
+    #!/bin/sh
+
+    set -ex
+
+    SLEEP_TIMEOUT=5
+    # we need to count seed instance here.
+    MINIMUM_CLUSTER_SIZE=4
```

Comment:
Should it be in relation with the replicas variable defined in stackanetes instead of being hardcoded?